Makefile

```diff
@@ -277,22 +278,22 @@ update-downloadcache: $(CACHE)
 # DEPS
 ######
 # Use the pyramid install dir as our indicator that dependencies are installed.
-$(PYRAMID): $(PY) $(CACHE)
+$(PYRAMID): $(PY) $(CACHE) requirements.txt
 	$(call PIP,requirements.txt)
+	@touch $(PYRAMID)
```

Comment:
PYRAMID is the pre-requisite for the target.  If the pre-req is not updated then that target will execute over and over.  It is not a canary.